packages/xod-func-tools/src/objects.js

```diff
@@ -143,3 +143,10 @@ export const sameKeysetBy = def(
     return true;
   }
 );
+
+// :: [StrMap [a]] -> StrMap [a]
+export const mergeAllWithConcat = R.reduce(R.mergeWith(R.concat), {});
+
+// ObjectOrArray :: [a] | StrMap ObjectOrArray
+// :: [StrMap ObjectOrArray] -> StrMap ObjectOrArray
+export const mergeAllDeepWithConcat = R.reduce(R.mergeDeepWith(R.concat), {});
```

Comment:
That's a _very_ specific function. And it's name basically matches the implementation.
Do we really need to bloat our API with it?